MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -42,8 +42,14 @@ public sealed class CreditCard
         ///  as provided by the payment processor.</param>
         /// <param name="token">A token uniquely identifying the card. This
         /// should not be the actual credit card number.</param>
-        /// <param name="was3dSecureSuccessful">Whether or not the 3DS check was
-        /// successful, as provided by the end user.</param>
+        /// <param name="was3dSecureSuccessful">Whether or not the 3D-Secure
+        /// verification (e.g. Safekey, SecureCode, Verified by Visa) was
+        /// successful, as provided by the end user. `true` if customer
+        /// verification was successful, or `false` if the customer failed
+        /// verification. If 3-D Secure verification was not used, was
+        /// unavailable, or resulted in an outcome other than success or
+        /// failure, do not include this successful, as provided by the end
```

Comment:
`do not include this successful...` seems like a copy+paste error

Reply:
💯